ice/metrics/gold_standards.py

```diff
@@ -138,7 +138,7 @@ def select_column_values(
     return df.loc[mask]
 
 
-_paper_dir = Path("/code/papers/")
+_paper_dir = Path(__file__).parent.parent.parent / "papers"
```

Comment:
I think this is worth extracting into `settings.PAPER_DIR` or something, both for deduplication and potential configurability.